Makefile

```diff
@@ -274,13 +274,15 @@ verify-extended: check-generate check format test-cov test-cov-clean test-integr
 #####################################################################
 
 kind-up: $(KIND)
+    # mkdir -m 775 -p $REPO_ROOT/dev/local-backupbuckets
```

Comment:
I think keeping this directory creation explicit is ok, so that it's not confusing in `kind-down` when it is deleted.